content/millennium/r4/financial/general/insurance-plan.md

```diff
@@ -36,11 +44,18 @@ Search for InsurancePlans that meet supplied query parameters:
 
 ### Parameters
 
-Name                  | Required?          | Type          | Description
-----------------------|--------------------|---------------|-------------------------------------------------------
-`_id`                 | This or `owned-by` | [`token`]     | The logical resource id associated with the resource.
-`owned-by`            | Yes                | [`reference`] | Organization associated with the InsurancePlan.
+Name       | Required      | Type          | Description
+-----------|---------------|---------------|-------------------------------------------------------
+`_id`      | Conditionally | [`token`]     | The logical resource ID associated with the resource. This parameter is required if the `owned-by` parameter is not used. Example: `12345`
+`owned-by` | Conditionally | [`reference`] | The organization associated with the insurance plan. This parameter is required if the `_id` parameter is not used. Example: `12345`
```

Comment:
since this is a reference type, the value should be `Organization/12345`. we accept just the Organization.id, but in general, I prefer to be as explicit as possible.

`owned-by=589783` and `owned-by=Organization/589783` are currently functionally equivalent. using a different resource type returns a 400 with `owned-by: expected resource type 'Organization'`

I'd also use a different number than the _id example